pxr/usdImaging/lib/usdImaging/delegate.cpp

```diff
@@ -234,7 +238,9 @@ UsdImagingDelegate::_AdapterLookup(UsdPrim const& prim, bool ignoreInstancing)
     //    threads.
 
     TfToken adapterKey;
-    if (!ignoreInstancing && prim.IsInstance()) {
+    if (_displayUnloadedPrimsWithBounds && !prim.IsLoaded()) {
```

Comment:
By putting this as the first "if" case, if your unloaded prim is a native instance, you won't get the benefit of instancing.  I think that's much simpler, and may be fine.

I'm curious what would happen if you swap those two cases, though.  By my read, the unloaded prim goes into the instance adapter, gets bound to a hydra instancer, and then the prototype population loop would end up right back in _AdapterLookup, but this time with "ignoreInstancing" set, and so it would populate the unloaded prim as a draw mode object.  If you fix the UsdPrimRange in instanceAdapter.cpp to use the new predicate, it might just work automagically...